lib/chef/knife/bootstrap.rb

```diff
@@ -417,7 +418,7 @@ def initialize(argv = [])
       # @return [String] Default bootstrap template
       def default_bootstrap_template
         if target_host.base_os == :windows
-          "windows-chef-client-msi"
+          "windows-#{Chef::Dist::CLIENT}-msi"
```

Comment:
I wasn't sure about this one but I went ahead and added it so we could discuss any implications I might have missed

Reply:
we might just rename those templates altogether.